### DISCUSSION TITLE
dietPi - apt upgrade / dietpi-update - keep docker running

### DISCUSSION THREAD

**Discussion by isarrider** [MichaIng/DietPi#6038] [Q&A] [answered]
Hi,

is there a way to leave docker running, if it itself wont be updated (or its dependencies)?

I have some containers running, that shouldn't be restarted...
But this way, I cant upgrade...
(either dietpi or apt...)

Thx,
Alex

**Comment by Joulinar** [comment, marked as answer]
> is there a way to leave docker running

Shortanswer: 
1. `yes`.

Long answer: 
1. open `dietpi-services`
2. select `docker`
3. switch `Include/Exclude` to `[excluded]`
4. done

**Reply by MichaIng** [reply]
Also run `systemctl enable docker` so that it is started at boot and restarted on package upgrades.

**Comment by isarrider** [comment]
awesome - thnak you!
